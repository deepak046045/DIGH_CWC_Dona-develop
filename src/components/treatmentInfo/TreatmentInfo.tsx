import React from 'react';
import layout from '@/scss/themes/layout.module.scss';
import styles from './TreatmentInfo.module.scss';

export default function TreatmentInfo() {
  return (
    <div className={`${styles.container} ${layout.gutter}`}>
      <div className={styles.item}>
        <h2 className={styles.heading}>Treatment Touch Points</h2>
        <p className={styles.details}>
          Throughout your treatment journey youâ€™ll have regular touch points or
          visits with your health care provider. In the beginning of treatment
          these occur more frequently as you work to understand how effective
          the treatment is and set yourself up for success with ongoing
          treatment.
        </p>
      </div>
      <div className={styles.item}>
        <h2 className={styles.heading}>Recommended Dosing</h2>
        <p className={styles.details}>
          The recommended dosing shows a clear path for timing of doses, what
          kind of dose it is, and how much to take. For Kinsula your doses are
          in the form of infusions.
        </p>
      </div>
    </div>
  );
}
