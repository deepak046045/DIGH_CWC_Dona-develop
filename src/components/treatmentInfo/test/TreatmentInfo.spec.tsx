import React from 'react';
import { render } from '@testing-library/react';
import TreatmentInfo from '../TreatmentInfo';

describe('TreatmentInfo', () => {
  it('should render page', () => {
    const { container } = render(<TreatmentInfo />);

    expect(container).toMatchSnapshot();
  });
});
