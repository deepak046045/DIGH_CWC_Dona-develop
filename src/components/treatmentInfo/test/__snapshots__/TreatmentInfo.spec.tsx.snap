// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreatmentInfo should render page 1`] = `
<div>
  <div
    class="container gutter"
  >
    <div
      class="item"
    >
      <h2
        class="heading"
      >
        Treatment Touch Points
      </h2>
      <p
        class="details"
      >
        Throughout your treatment journey youâ€™ll have regular touch points or visits with your health care provider. In the beginning of treatment these occur more frequently as you work to understand how effective the treatment is and set yourself up for success with ongoing treatment.
      </p>
    </div>
    <div
      class="item"
    >
      <h2
        class="heading"
      >
        Recommended Dosing
      </h2>
      <p
        class="details"
      >
        The recommended dosing shows a clear path for timing of doses, what kind of dose it is, and how much to take. For Kinsula your doses are in the form of infusions.
      </p>
    </div>
  </div>
</div>
`;
